components/infieldbutton/themes/spectrum-two.css

```diff
@@ -27,5 +27,9 @@
 		--spectrum-infield-button-background-color-hover: var(--spectrum-gray-200);
 		--spectrum-infield-button-background-color-down: var(--spectrum-gray-200);
 		--spectrum-infield-button-background-color-key-focus: var(--spectrum-gray-200);
+
+		&:disabled {
+			--spectrum-infield-button-border-color: var(--spectrum-gray-300);
```

Comment:
This probably could be set to `disabled-border-color` if we wanted. I think that's also `gray-300`.